pandas/compat/numpy/function.py

```diff
@@ -203,26 +206,26 @@ def validate_cum_func_with_skipna(skipna, args, kwargs, name):
 validate_max = CompatValidator(MINMAX_DEFAULTS, fname='max',
                                method='both', max_fname_arg_count=1)
 
-RESHAPE_DEFAULTS = dict(order='C')
+RESHAPE_DEFAULTS = dict(order='C')  # type: Dict[str, str]
 validate_reshape = CompatValidator(RESHAPE_DEFAULTS, fname='reshape',
                                    method='both', max_fname_arg_count=1)
 
-REPEAT_DEFAULTS = dict(axis=None)
+REPEAT_DEFAULTS = dict(axis=None)  # type: Dict[str, None]
 validate_repeat = CompatValidator(REPEAT_DEFAULTS, fname='repeat',
                                   method='both', max_fname_arg_count=1)
 
-ROUND_DEFAULTS = dict(out=None)
+ROUND_DEFAULTS = dict(out=None)  # type: Dict[str, None]
 validate_round = CompatValidator(ROUND_DEFAULTS, fname='round',
                                  method='both', max_fname_arg_count=1)
 
-SORT_DEFAULTS = OrderedDict()
+SORT_DEFAULTS = OrderedDict()  # type: OrderedDict[str, Union[int, str, None]]
```

Comment:
Use Optional instead of Union[....,. None]

Reply:
This comment is applicable in a few other places

Reply:
@WillAyd , `Optional` takes only one argument, so instead of

    Union[X, Y, None]

Can I use optional like this 👇 

    Optional[Union[X, Y]]

I am a bit busy on weekdays, I will fix it if I get some time, otherwise it will be on saturday(friday for you), sorry about that.

Reply:
@WillAyd all done, turns out I found some free time. 😄